.tests/integration/config/config.yaml

```diff
@@ -24,9 +24,12 @@ filtering_variant_filtration:
   read_depth: 100
   container: "docker://quay.io/biocontainers/pysam:0.19.1--py310hd89ff4b_0"
```

Comment:
would the common container work? It contains pysam 0.18

Reply:
I believe it will, I haven't found the specific requirements in the original repo, but I think pysam version was much lower than 0.18. I'll test with the common container and will see what happens.

Reply:
I tested it and it works fine.